storage/metricsstore/interface.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metricsstore
+
+import (
+	"context"
+	"time"
+
+	"github.com/jaegertracing/jaeger/proto-gen/api_v2"
```

Comment:
perhaps we should move the gen-types under api_v2/metrics?

Reply:
+1